tests/integration/testlib/nmplugin.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+import re
+
+from . import cmdlib
+
+
+SYSTEMCTL_RESTART_NM_CMD = ("systemctl", "restart", "NetworkManager")
+
+
+@contextmanager
+def disable_nm_plugin(plugin):
+    ret, rpm_output, _ = cmdlib.exec_cmd(
+        ("rpm", "-ql", f"NetworkManager-{plugin}")
+    )
+    if not ret:
+        lib_path = re.search(r"[A-Za-z0-9-/.]*.so", rpm_output).group()
```

Comment:
You can use something like `[l for l in rpm_output.split() if l.endswith(b".so")][0]` to avoid the regular expression which  is harder to understand IMHO. (Meant to mention this before but I could not find an old comment about this).

Reply:
Oh! Thanks for this one. I am glad to drop the regexp :-)

Reply:
Done, thanks!